### PR TITLE
add GPS+RTCM output mode for logging (for PPK)

### PR DESCRIPTION
**Pull request by bkueng**

### CODE REVIEW

**Review by LorenzMeier — APPROVED** [review]

**Comment by LorenzMeier** [comment]
Thanks!

File: src/ubx.cpp [hunk 1]
```diff
@@ -83,9 +83,7 @@ GPSDriverUBX::GPSDriverUBX(Interface gpsInterface, GPSCallbackPtr callback, void
 
 GPSDriverUBX::~GPSDriverUBX()
 {
-	if (_rtcm_parsing) {
-		delete (_rtcm_parsing);
-	}
+	delete _rtcm_parsing;
```
Comment (lukegluke) [inline src/ubx.cpp:86]: @bkueng it looks like the bug. `_rtcm_parsing` is not created in simple GPS OutputMode. It is not a good idea to remove `_rtcm_parsing` null checking.

Comment (bkueng) [reply src/ubx.cpp:86]: Passing `nullptr` to `delete` is valid

Comment (lukegluke) [reply src/ubx.cpp:86]: Yeah, you are right, I missed it. Then it also could be done in other drivers, too :)

Comment (bkueng) [reply src/ubx.cpp:86]: Yes, definitely

**Pull request merged**